mmdet/core/bbox/assigners/atss_cost_assigner.py

```diff
@@ -0,0 +1,281 @@
+import torch
+
+from ..builder import BBOX_ASSIGNERS
+from ..iou_calculators import build_iou_calculator
+from .assign_result import AssignResult
+from .base_assigner import BaseAssigner
+
+def diou_loss(pred, target, eps=1e-7):
```

Comment:
Can you add `diou` calculation function to [bbox_overlaps](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/bbox/iou_calculators/iou2d_calculator.py#L75), it will be better to use